src/w3d/renderer/motchan.h

```diff
@@ -121,17 +121,17 @@ class TimeCodedMotionChannelClass : W3DMPO
     virtual ~TimeCodedMotionChannelClass() override;
     void Free();
     bool Load_W3D(ChunkLoadClass &cload);
-    int Get_Type() { return m_type; }
-    int Get_Pivot() { return m_pivotIdx; }
+    int Get_Type() const { return m_type; }
+    int Get_Pivot() const { return m_pivotIdx; }
     void Get_Vector(float frame, float *setvec);
     Quaternion Get_Quat_Vector(float frame_idx);
-    void Set_Identity(float *setvec);
-    unsigned long Get_Index(unsigned int timecode);
-    unsigned long Binary_Search_Index(unsigned int timecode);
+    void Set_Identity(float *setvec) const;
+    unsigned int Get_Index(unsigned int timecode);
+    unsigned int Binary_Search_Index(unsigned int timecode) const;
```

Comment:
Confirmed in the original as returning unsigned long.
Also this function isn't const in the original.